lib/widgets/file_drop_overlay.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/material.dart';
+
+class FileDropOverlay extends StatelessWidget {
+  final Widget? graphic;
+  final String? title;
+  final String? subTitle;
```

Comment:
Rename to `subtitle`, as in https://api.flutter.dev/flutter/material/ListTile/subtitle.html